### DISCUSSION TITLE
Video in Notification does not play.

### DISCUSSION THREAD

**Discussion by wild-a** [koush/scrypted#1086] [Q&A]
Homekit Secure Video is working fine, I am getting all the notifications, video quality is good.
However, when I receive a notification and "tap and hold" it, it should play the video that was captured for that motion, but it does not play anything.

**Comment by wild-a** [comment]
![image](https://github.com/koush/scrypted/assets/74679016/741b4307-bb21-45ea-a9e8-f48c17ae9139)

This is what it looks like when I view the notification, it appears to be a still image. Normally, it should start playing the clip that was recorded.

**Comment by koush** [comment]
apple issue.

**Reply by wild-a** [reply]
I don't believe so. I was running the same camera in Homebridge before setting up Scrypted, and it was working fine.

**Comment by tellok** [comment]
same here 😢

**Reply by wild-a** [reply]
Were you able to find a solution?

**Reply by tellok** [reply]
It turns out to be a network problem, I improved the wifi connection strength, and it's ok now.
